modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordsDisplayContext.java

```diff
@@ -173,7 +174,8 @@ public String getColumnName(DDMFormField ddmFormField) {
 	}
 
 	public String getColumnValue(
-		DDMFormField ddmFormField, List<DDMFormFieldValue> ddmFormFieldValues) {
+		DDMFormField ddmFormField, List<DDMFormFieldValue> ddmFormFieldValues,
+		String fieldName) {
```

Comment:
@marcelabc, can we rename `fieldName` to `ddmFormFieldName` to keep consistency with existing back-end code?